components/frontend-react/src/containers/QueuedFileList.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { deleteDownload } from '../actions/downloadActions';
 
 //MUI imports
 import { withStyles } from '@material-ui/core/styles';
```

Comment:
Use makeStyles hooks way or withStyles HOC only. I think you have used makeStyles in some places, right?

Reply:
Yes Chathu, I have used withStyles for class components and makeStyles for functional components. As makeStyles is a React Hook while withStyles is a HOC, so I was unable to use makeStyles in class componennts.